src/mvAppItem.cpp

```diff
@@ -4086,13 +4198,32 @@ DearPyGui::GetEntityParser(mvAppItemType type)
             MV_PARSER_ARG_SHOW)
         );
 
-        args.push_back({ mvPyDataType::Double, "x" });
-        args.push_back({ mvPyDataType::Double, "y" });
-        args.push_back({ mvPyDataType::Integer, "x_offset", mvArgType::KEYWORD_ARG });
-        args.push_back({ mvPyDataType::Integer, "y_offset", mvArgType::KEYWORD_ARG });
+        args.push_back({ mvPyDataType::DoubleList, "x" });
+        args.push_back({ mvPyDataType::DoubleList, "y" });
+        args.push_back({ mvPyDataType::Integer, "x_offset", mvArgType::DEPRECATED_KEYWORD_ARG, "0", "Old way to set x offset of the label. Use `offset` argument instead.", "See the new offset argument." });
```

Comment:
`x_offset`, `y_offset`: These are not parsed in `mvPlotting.cpp`, and do not really work. Earlier I suggested to declare them deprecated but still accept them so that they _work like in the old version_ (in order not to break existing code).

Reply:
Scratch this. I forgot that `add_text_point` actually never worked in the old version due to a wrong parser :joy:. (and that mess probably caused false memories of me already pointing to these args :)).

No worries, let's leave them as they are now - deprecated in `mvAppItem.cpp`, ignored in `mvPlotting.cpp`.